Library/Homebrew/build_environment.rb

```diff
@@ -59,7 +59,7 @@ def dump_build_env(env, f = $stdout)
       s = "#{key}: #{value}"
```

Comment:
```suggestion
      s = +"#{key}: #{value}"
```

Make the string mutable instead.

Reply:
If that's what you'd prefer, I can do it. I indicated in a [previous comment](https://github.com/Homebrew/brew/pull/6080#issuecomment-488043199) that I opted not to do that because it results in unnecessary String dupes, effectively undermining having frozen string literals enabled in the first place.